src/app/pages/profile/profile.page.ts

```diff
@@ -17,14 +17,10 @@ export class ProfilePage implements OnInit {
 
     this.route.data.subscribe((data) => {
       if (this.router.getCurrentNavigation().extras.state) {
-        console.log("save passed data");
```

Comment:
Removing the _console.log_ lines because they've each fulfilled their purpose of ensuring the code does as intended?

Reply:
We should be using unit tests to actually test functions. We should be using a logging framework like Winston if we want to actually log things. This just clutters the console, needlessly. Additionally, using the dev toolkit provided by Chrome or Firefox should be enough to inspect what data is available at any given time.